cowprotocol/accounting/rewards/mainnet/mainnet_dashboard_query_2510345.sql

```diff
@@ -0,0 +1,222 @@
+with
+
+block_range as (
+    select * from "query_3333356(start_time='{{start_time}}',end_time='{{end_time}}')"
+),
+
+results_per_tx as (
+    select * from "query_3427730(start_time='{{start_time}}',end_time='{{end_time}}',cte_name='results_per_tx')"
+
+),
+
+solver_slippage as (
+    select
+        solver_address as solver,
+        sum(eth_slippage_wei) * 1.0 / pow(10, 18) as slippage
+    from results_per_tx
+    group by solver_address
+),
+
+named_results as (
+    select * from "1541516(end_time='{{end_time}}',cte_name='named_results')"
+),
+-- end SLIPPAGE
+
+-- BEGIN SOLVER REWARDS: https://dune.com/queries/2283297
+batch_rewards_temp as (
+    select  --noqa: ST06
+        rbr.block_deadline,
+        rbr.block_number,  -- Null here means the settlement did not occur.
+        from_hex(rbr.solver) as winning_solver,
+        from_hex(rbr.tx_hash) as tx_hash,
+        -- Unpacking the data
+        cast(cast(rbr.data.winning_score as varchar) as int256) as winning_score,  -- noqa: RF01
+        cast(cast(rbr.data.reference_score as varchar) as int256) as reference_score,  -- noqa: RF01
+        cast(cast(rbr.data.surplus as varchar) as int256) as surplus,  -- noqa: RF01
+        cast(cast(rbr.data.fee as varchar) as int256) as fee,  -- noqa: RF01
+        cast(cast(rbr.data.execution_cost as varchar) as int256) as execution_cost,  -- noqa: RF01
+        cast(cast(rbr.data.uncapped_payment_eth as varchar) as int256) as uncapped_payment_eth,  -- noqa: RF01
+        cast(cast(rbr.data.capped_payment as varchar) as int256) as capped_payment,  -- noqa: RF01
+        transform(rbr.data.participating_solvers, x -> from_hex(x)) as participating_solvers,  -- noqa: RF01
+        cardinality(rbr.data.participating_solvers) as num_participants  -- noqa: RF01
+    from cowswap.raw_batch_rewards as rbr
+    where
+        rbr.block_deadline > (select start_block from block_range)
+        and rbr.block_deadline <= (select end_block from block_range)
+),
+
+batch_rewards as (
+    select
+        block_deadline,
+        block_number,
+        winning_solver,
+        tx_hash,
+        winning_score,
+        reference_score,
+        surplus,
+        fee,
+        execution_cost,
+        capped_payment,
+        participating_solvers,
+        num_participants
+    from batch_rewards_temp
+),
```

Comment:
These two queries can be combined into
```suggestion
batch_rewards as (
    select
        from_hex(rbr.solver) as winning_solver,
        -- Unpacking the data
        cast(cast(rbr.data.capped_payment as varchar) as int256) as capped_payment,  -- noqa: RF01
        cast(cast(rbr.data.fee as varchar) as int256) as fee,  -- noqa: RF01
        cast(cast(rbr.data.execution_cost as varchar) as int256) as execution_cost,  -- noqa: RF01
    from cowswap.raw_batch_rewards as rbr
    where
        rbr.block_deadline > (select start_block from block_range)
        and rbr.block_deadline <= (select end_block from block_range)
),
```
since we only need `winning_solver`, `capped_payment`, `fee`, `execution_costs` below.

Reply:
Good point. Pushed a change.